.github/workflows/fork-to-branch.yaml

```diff
@@ -0,0 +1,54 @@
+name: FORK-TO-BRANCH
+
+on:
+  workflow_dispatch:
+    inputs:
+      source-branch:
+        type: string
+        description: The name of the branch from forked-repo.
+        required: true
+      contributor-name:
+        type: string
+        description: write the contributor github username
+        required: true
```

Comment:
This suppose to run automatically when a fork PR is opened and update the branch when the PR is updated